eval/action.yml

```diff
@@ -0,0 +1,66 @@
+name: "Sandard Eval"
+description: "Evaluate Standard based projects"
+
+inputs:
+  key_path:
+    description: "The path to place the Nix secret Key"
+    required: true
+    default: "${{ runner.temp }}/.nix-key"
+  github_pat:
+    description: "GitHub Token for Accessing Private Repos with Nix"
+    required: true
+    default: ${{ github.token }}
+  nix_key:
+    description: "Secret Key used for Nix Package Signing"
+    required: false
+  s3_key:
+    description: "Key to upload to S3 Buckets with Nix"
+    required: false
+  s3_id:
+    description: "ID to upload to S3 Buckets with Nix"
+    required: false
+  cache:
+    description: "Nix cache to optionally upload artifacts to"
+    # auto refers to the local nix store, so nothing is uploaded by default
+    default: auto
+    required: false
+
+
+runs:
+  using: "composite"
+  steps:
+    - name: Emplace Secrets
```

Comment:
Shouldn't that be run on the condition of any meaningful secret being configured?

Reply:
Implicitly it already is because the script has guards to not do anything if they don't exist.

Reply:
That's the work part. But what about the UX representation?

A thing that does "nothing" doesn't need a placeholder that would actively confuse / mislead people into beleiving that secrets are handled.